hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -180,9 +181,14 @@ class WorkflowParams(param.Parameterized):
                                       "metrics to AzureML. Both intermediate validation metrics and final test results"
                                       "will be recorded. You need to have an AzureML workspace config.json file "
                                       "and will be asked for interactive authentication.")
+    different_seeds: int = param.Integer(default=0, bounds=(0, None),
```

Comment:
Since we have an arg called hyperdrive too, should we make it clear here not to try to set both `different_seeds` and `hyperdrive`?

Reply:
good point, added.